lib/internal/debugger/inspect_repl.js

```diff
@@ -118,16 +118,12 @@ function extractFunctionName(description) {
   return fnNameMatch ? `: ${fnNameMatch[1]}` : '';
 }
 
-const {
-  builtinIds: PUBLIC_BUILTINS,
-} = internalBinding('builtins');
-const NATIVES = internalBinding('natives');
+const { builtinIds } = internalBinding('builtins');
 function isNativeUrl(url) {
   url = SideEffectFreeRegExpPrototypeSymbolReplace(/\.js$/, url, '');
 
   return StringPrototypeStartsWith(url, 'node:internal/') ||
-         ArrayPrototypeIncludes(PUBLIC_BUILTINS, url) ||
-         url in NATIVES || url === 'bootstrap_node';
```

Comment:
The original condition was already redundant, `builtinIds` actually contains both public and internal IDs.